packages/flutter/lib/src/cupertino/desktop_text_selection.dart

```diff
@@ -70,7 +70,11 @@ class CupertinoDesktopTextSelectionControls extends TextSelectionControls {
 
   /// Gets the position for the text selection handles, but desktop has none.
   @override
-  Offset getHandleAnchor(TextSelectionHandleType type, double textLineHeight) {
+  Offset getHandleAnchor(
+    TextSelectionHandleType type,
+    double textLineHeight, {
+    required double cursorWidth,
```

Comment:
This could be a breaking change. Maybe make it not required and include the default 2.0 value here if it's not provided?